### DISCUSSION TITLE
Rotateable node

### DISCUSSION THREAD

**Discussion by psaurer-prodyna** [Foblex/f-flow#129] [Feature Request] [answered]
### Description

Hello!

First of all, thanks for this library and great documentation.

I would have a usecase where the nodes should be possible to be rotated. I am not sure if it is maybe already possible. Is there any way that nodes can be created with an angle or can be rotated? Or some workaround with custom style per node?

Thanks in advance!

**Comment by siarheihuzarevich** [comment, marked as answer]
Hi! Thanks a lot for the kind words and for your interest in the library 🙌

Support for node rotation was added in version 17.2.4.
You can now use the built-in fRotateHandle directive to rotate a node using a rotation handle.

Here’s a live example showing how it works:
🔗 [https://flow.foblex.com/examples/rotate-handle](https://flow.foblex.com/examples/rotate-handle)

You can also set the initial angle using the fNodeAngle input or update it programmatically.
Let me know if you need help setting it up!

**Reply by psaurer-prodyna** [reply]
Hi, thank you so much for this feature, looks great. Will let you know if I need some help!
